go.mod

```diff
@@ -19,8 +19,6 @@ require (
 	github.com/coreos/go-systemd v0.0.0-20190719114852-fd7a80b32e1f
 	github.com/davecgh/go-spew v1.1.1
 	github.com/decred/dcrd/dcrec/secp256k1/v4 v4.3.0
-	github.com/go-logr/logr v1.4.2 // indirect
```

Comment:
isn't this done by go mod automatically ? or was is done automatically but I wonder why it has its own commit